airflow/configuration.py

```diff
@@ -468,12 +468,19 @@ class AirflowConfigParser(ConfigParser):
         ('celery', 'celery_result_backend')
     }
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, default_config=None, *args, **kwargs):
```

Comment:
Ah, yes, that's because keyword only arguments were added in Python 3. My mistake.